Docs/user_guide/data_loading_and_saving.md

```diff
@@ -62,11 +62,8 @@ Multiple nodes can be exported at once by placing them into a folder and then by
 
 ### On use of LPS/RAS coordinate systems
 
-DICOM and most medical imaging software uses the **LPS coordinate system** for storing all data. Axis directions of this coordinate system are patient left, posterior, and superior. Unit is millimeters. The choice of origin is arbitrary because only relative differences have meaning, so there is no universal standard, but it is often set to some geometric center of the imaging system, or it is chosen to be near the center of an object of interest.
-
-**RAS coordinate system** is almost the same as the LPS coordinate system, except the axis directions are patient right, anterior, superior, i.e., the sign of the first two coordinates is inverted. Both LPS and RAS were in wide use in the early 2000s when development of Slicer was started and Slicer developers chose to use the RAS coordinate system.  Historically scans by GE equipment used RAS while Siemens and others used LPS.  Since several GE researchers were early contributors to Slicer, RAS was adopted for the internal representation.
-
-Slicer still uses RAS coordinate system for storing coordinate values internally for all data types, but for compatibility with other software, it assumes that all data in files are stored in LPS coordinate system (unless the coordinate system in the file is explicitly stated to be RAS). To achieve this, whenever Slicer reads or writes a file, it may need to flip the sign of the first two coordinate axes to convert the data to RAS coordinate system. 
+See the [Coordinate system convention in Slicer](../user_guide/coordinate_systems.md#coordinate-system-convention-in-slicer)
+section to read about Slicer's coordinate system convention.
```

Comment:
```suggestion
DICOM and medical imaging software use the LPS (Left, Posterior, Superior) coordinate system, while Slicer's internal representation employs RAS (Right, Anterior, Superior). For file compatibility, Slicer assumes data in files are in LPS coordinates and may flip the first two axes during read or write operations.

To learn more, see [Image and anatomical coordinate systems](../user_guide/coordinate_systems.md) and [Coordinate system convention in Slicer](../user_guide/coordinate_systems.md#coordinate-system-convention-in-slicer)
```

Here is a more explicit summary.

@pieper @lassoan  What do you think linking to `Coordinate system convention in Slicer`?

Reply:
Done in [17cea93](https://github.com/Slicer/Slicer/commit/17cea93d3bc6b3279333b3f3a71728394a55bd7c).